verification-annotations/src/seahorn.rs

```diff
@@ -38,7 +38,7 @@ pub fn abort() -> ! {
 /// Any paths found must satisfy this assumption.
 pub fn assume(pred: bool) {
     if ! pred {
-        unsafe { __VERIFIER_assume(0); }
+        unsafe { __VERIFIER_assume(pred as i32); }
```

Comment:
Is there an efficiency difference in SeaHorn between

"if !pred { unsafe { __VERIFIER_assume(0)} }"

and 

"__VERIFIER_assume(pred as i32)"

I think they mean the same but the latter maybe creates less branches/merges?

Reply:
Sorry I missed reading the `if`. This is not needed. 

thanks